AzureBot/Dialogs/ActionDialog.cs

```diff
@@ -35,18 +35,29 @@ public class ActionDialog : LuisDialog<string>
 
             context.Wait(MessageReceived);
         }
-
+        [LuisIntent("Hello")]
         [LuisIntent("Help")]
         public async Task Help(IDialogContext context, LuisResult result)
         {
-            string message = "Hello! You can use the Azure Bot to: \n";
+            string message = "";
+            var accessToken = await context.GetAccessToken(resourceId.Value);
+            if (string.IsNullOrEmpty(accessToken))
+            {
+                message += $"Welcome to the Azure Bot!\n\n";

```

Comment:
Let's use a more personal voice here :) and just go with "Hello!" for the first time.